app/components/AnalystDashboard/AllDashboard.tsx

```diff
@@ -350,6 +352,11 @@ const AllDashboardTable: React.FC<Props> = ({ query }) => {
     filterFns: {
       filterNumber,
     },
+    renderTopToolbarCustomActions: () => (
+      <Box>
```

Comment:
I'm wondering why the <Box> around the ClearFilters component - as far as I know all it does is add a <div> around it if there's no styles attached.

Reply:
yeah, removed